packages/salesforcedx-vscode-core/src/commands/forceApexLogGet.ts

```diff
@@ -208,14 +214,95 @@ export class ForceApexLogList {
   }
 }
 
-const workspaceChecker = new SfdxWorkspaceChecker();
-const parameterGatherer = new LogFileSelector();
+export class LogGetGatherer implements ParametersGatherer<{ Id: string }> {
```

Comment:
re: earlier conversation, we should be able to use `LogFileSelector` for both implementations